CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add feature to make includes, literalincludes, and figures clickable from within the text editor (DOCSP-5967).
```

Comment:
Well, more specifically, in the snooty parser you added the `textDocument/resolve` RPC endpoint to return an artifact's corresponding source file path relative to the project's root.